tests/sanity/tests/model/tracker/issues-page.ts

```diff
@@ -211,9 +210,28 @@ export class IssuesPage extends CommonTrackerPage {
   }
 
   async checkAllIssuesByPriority (priorityName: string): Promise<void> {
-    for await (const locator of iterateLocator(this.issuesList)) {
-      const href = await locator.locator('div.priority-container use').getAttribute('href')
-      expect(href).toContain(priorityName)
+    let retry = 150
+    while (retry > 0) {
```

Comment:
there is built in `expect.toPass` method for this:
https://playwright.dev/docs/test-assertions#expecttopass

```
await expect(async () => {
...
}).toPass({
  intervals: [100],
  timeout: 30000
})
```

Reply:
Nice:-)